appOPHD/UI/DetailMap.cpp

```diff
@@ -134,9 +131,12 @@ void DetailMap::draw() const
 			// Draw a beacon on an unoccupied tile with a mine
 			if (tile.mine() != nullptr && !tile.thing())
 			{
-				uint8_t glow = static_cast<uint8_t>(120 + std::sin(throbTimer.tick() / ThrobSpeed) * 57);
-				renderer.drawImage(mMineBeacon, position + NAS2D::Vector{0, -64});
-				renderer.drawSubImage(mMineBeacon, position + NAS2D::Vector{59, 15}, NAS2D::Rectangle<int>{{59, 79}, {10, 7}}, NAS2D::Color{glow, glow, glow});
+				constexpr NAS2D::Vector<int> offsetInTile{0, -64};
+				constexpr NAS2D::Vector<int> lightOffsetInTile{59, 15};
+				constexpr NAS2D::Rectangle<int> lightSubImageRect{{59, 79}, {10, 7}};
+				uint8_t glow = DetailMap::glow();
+				renderer.drawImage(mMineBeacon, position + offsetInTile);
+				renderer.drawSubImage(mMineBeacon, position + lightOffsetInTile, lightSubImageRect, NAS2D::Color{glow, glow, glow});
```

Comment:
I like having the glow intensity function extracted and called. That makes this code much easier to read.

I'm wondering if the glow intensity function could be wrapped in a `glowColor` function, that returns a fully constructed `NAS2D::Color` object. It would also give a chance to name that parameter in the later call. I'd probably want to use `auto` for the variable type in that case.